crates/fuel-core/src/state/in_memory/memory_store.rs

```
@@ -161,12 +159,25 @@ where
         Ok(())
     }
 
+    fn view_at_height(
+        &self,
+        _: &Description::Height,
+    ) -> StorageResult<KeyValueView<Self::Column>> {
+        // TODO: https://github.com/FuelLabs/fuel-core/issues/1995
+        unimplemented!("The historical view is not implemented for `MemoryStore`")
```

Should we return an error here instead?